### ISSUE TITLE
impl not generating stubs from interface

### ISSUE THREAD

**Issue by danf0rth** [Microsoft/vscode-go#1769]
`Cannot stub interface: unrecognized interface: user.UserRepository`

`go version go1.10.2 linux/amd64`
`Extension version 0.6.84`, and i also tried `0.6.85-Beta1` 

Steps to Reproduce:
1. Call command `Go: Generate interface stubs`
2. Type `m MySQLUserRepository user.UserRepository`
3. Error occurred: `Cannot stub interface: unrecognized interface: user.UserRepository`

File structure is similar to [this](https://github.com/bxcodec/go-clean-arch/tree/master/article) repo, except all located in `internal` folder of package.

If I run in terminal `impl 'm MySQLUserRepository' user.UserRepository`, it works:
```
func (m MySQLUserRepository) GetByID(ctx context.Context, id int64) (*model.User, error) {
        panic("not implemented")
}

func (m MySQLUserRepository) Update(ctx context.Context, user *model.User) (*model.User, error) {
        panic("not implemented")
}

func (m MySQLUserRepository) Store(ctx context.Context, user *model.User) (*model.User, error) {
        panic("not implemented")
}

func (m MySQLUserRepository) Delete(ctx context.Context, user *model.User) (bool, error) {
        panic("not implemented")
}
```

**Comment by ramya-rao-a** [comment]
Can you try running `https://'github.com/josharian/impl` from the terminal and see if it is able to generate the stubs as expected? If not, then please log this issue at https://github.com/josharian/impl

**Comment by danf0rth** [comment]
As i said before, if i run command in terminal `impl 'm MySQLUserRepository' user.UserRepository`, it works. It looks like vscode run the impl in wrong directory.
![screenshot_2018-07-17_07-44-26](https://user-images.githubusercontent.com/14220891/42796934-70d00b82-8995-11e8-8fe8-5df7ef243241.png)

**Comment by ramya-rao-a** [comment]
> As i said before,

Sorry, I totally missed that. You are right, passing the cwd when spawning the process that runs `impl` fixes this issue. The fix will be out in the next update which should be in a few days

**Comment by danf0rth** [comment]
@ramya-rao-a, yes, problem occurs when the interface located in the `internal` directory

**Comment by ramya-rao-a** [comment]
The fix for this issue is now out in the latest update (0.6.85) to the Go extension.
